instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/auto/instrumentation/awssdk/v1_11/AwsSdkClientTracer.java

```diff
@@ -125,4 +125,14 @@ protected String requestHeader(Request<?> request, String name) {
   protected String responseHeader(Response<?> response, String name) {
     return response.getHttpResponse().getHeaders().get(name);
   }
+
+  @Override
+  protected Setter<Request<?>> getSetter() {
+    return null;
```

Comment:
Why is this `null`?

Reply:
There is no SETTER.

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/893#discussion_r466131521

Reply:
I don't understand. Do you want to say that one cannot inject new headers into aws sdk `Request`? And we cannot propagate context with it?

Reply:
aws sdk doesn't need to inject headers into its request.  @trask might know the reason why?

Reply:
my understanding is that no need to inject headers, since requests are all going to backend AWS services

Reply:
Then should it be `HttpClientTracer`? It has `startScope` method which will throw an exception if it gets null setter. So everybody has to remember not to call `startScope` for `AwsSdkClientTracer` which extends `HttpClientTracer`

Reply:
yeah, there's lots of stuff in aws-sdk instrumentation that doesn't follow the normal `Tracer` pattern so this may make sense.

I'll open a separate issue for this, so we can move this PR forward.

Reply:
Opened #944